lib/web/join_tokens.go

```diff
@@ -285,9 +301,9 @@ func getJoinScript(settings scriptSettings, m nodeAPIGetter) (string, error) {
 	var buf bytes.Buffer
 	// If app install mode is requested but parameters are blank for some reason,
 	// we need to return an error.
-	if settings.appInstallMode == true {
+	if settings.appInstallMode {
 		if errs := validation.IsDNS1035Label(settings.appName); len(errs) > 0 {
-			return "", trace.BadParameter("appName %q must be a valid DNS subdomain: https://gravitational.com/teleport/docs/application-access/#application-name", settings.appName)
+			return "", trace.BadParameter("appName %q must be a valid DNS subdomain: https://goteleport.com/docs/application-access/guides/connecting-apps/#application-name", settings.appName)
```

Comment:
As an aside, at some point we really ought to seriously consider getting a `teleport.link/abc123` short domain/link setup that we can use for arbitrary redirects from error messages to docs, just so we don't have to grep the codebase and fix domains/paths in future (or forget to...)